From 48466b32801ec7eb1f472267afe7940c92516210 Mon Sep 17 00:00:00 2001
From: Wenbinc-Bin <wenbin.chen@intel.com>
Date: Mon, 30 Nov 2020 13:30:49 +0800
Subject: [PATCH 53/82] libavcodec/qsvenc.c: add max_frame_size support for
 hevc_qsv

The max_frame_size parameter is set only when codec is h264. Now I add hevc in that conditional statement.

Signed-off-by: Wenbin CHEN <wenbin.chen@intel.com>
---
 libavcodec/qsvenc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libavcodec/qsvenc.c b/libavcodec/qsvenc.c
index 6f454b675c21..f5c4cc491a86 100644
--- a/libavcodec/qsvenc.c
+++ b/libavcodec/qsvenc.c
@@ -692,7 +692,7 @@ static int init_video_param(AVCodecContext *avctx, QSVEncContext *q)
         q->extparam_internal[q->nb_extparam_internal++] = (mfxExtBuffer *)&q->extco;
 
 #if QSV_HAVE_CO2
-        if (avctx->codec_id == AV_CODEC_ID_H264) {
+        if (avctx->codec_id == AV_CODEC_ID_H264 || avctx->codec_id == AV_CODEC_ID_HEVC) {
             if (q->int_ref_type >= 0)
                 q->extco2.IntRefType = q->int_ref_type;
             if (q->int_ref_cycle_size >= 0)
-- 
2.25.4

